File: Praktikum 2 C/PRAK204-2510817220002-AishaZwarintyanda.c
#include <stdio.h>
#include <string.h>

int main(){
    float jari;
    float tinggi;

    printf("Masukkan nilai jari-jari : ");
    scanf("%f", &jari);

    printf("Masukkan nilai tinggi : ");
    scanf("%f", &tinggi);

    float volume = (22.0 / 7.0) * (jari * jari) * tinggi;
    float luas = 2 * (22.0 / 7.0) * jari * (jari + tinggi);
    float keliling = 2 * (22.0 / 7.0) * jari;

    printf("Volume = %.2f\nLuas = %.2f\n Keliling = %.2f", volume, luas, keliling);
    return 0;
}

File: Praktikum 2 Python/PRAK204-2510817220002-AishaZwarintyanda.py
jari = float(input("Masukkan nilai jari-jari : "))
tinggi = float(input("Masukkan nilai tinggi : "))

volume = (22.0 / 7.0) * (jari * jari) * tinggi
luas = 2 * (22.0 / 7.0) * jari * (jari + tinggi)
keliling = 2 * (22.0 / 7.0) * jari

print(f"Volume = {volume:.2f}")
print(f"Luas = {luas:.2f}")
print(f"Keliling = {keliling:.2f}")